ReactSkia/sdk/NotificationCenter.h

```diff
@@ -0,0 +1,129 @@
+//
+// Copyright (c) 2014 Sean Farrell
```

Comment:
Would like to know more details about where the library coming from.

Reply:
The code was taken from https://gist.github.com/martinfinke/a636dcddbcf112344b59
Apart from renaming of module, have made the Listeners, mutex, and last_listener shared/usable across threads.